src/SymbolCollector.Server/Program.cs

```diff
@@ -73,7 +73,8 @@ public static async Task<int> Main(string[] args)
                     o.Dsn = "https://2262a4fa0a6d409c848908ec90c3c5b4@sentry.io/1886021";
                     o.ExperimentalMetrics = new ExperimentalMetricsOptions
                     {
-                        EnableCodeLocations = true
+                        EnableCodeLocations = true,
+                        CaptureSystemDiagnosticsMeters = BuiltInSystemDiagnosticsMeters.All
```

Comment:
@jamescrosswell @bitsandfoxes adding this explicitly if u decide to default to nothing but also to be explicit.

Lets dogfood!